src/main/java/com/odysseusinc/arachne/datanode/service/impl/AchillesServiceImpl.java

```diff
@@ -319,75 +324,75 @@ public void pullAchillesData(DataSource dataSource) {
             ExecutorService executorService = Executors.newCachedThreadPool(threadFactory);
             try {
                 List<Callable<String>> tasks = new LinkedList<>();
-                tasks.add(achillesTask("Heel", () -> runAchillesQuery(dataSource, ACHILLES_HEEL_SQL, tempDir.resolve("achillesheel.json"),
+                tasks.add(achillesTask("Heel", () -> runAchillesQuery(job.getDataSource(), ACHILLES_HEEL_SQL, tempDir.resolve("achillesheel.json"),
                         AchillesProcessors.achillesHeel())));
                 tasks.add(achillesTask("DrugEra", () -> {
                     List<Integer> drugEraConcepts = new LinkedList<>();
-                    Integer result = runAchillesQuery(dataSource, DRUG_ERA_SQL, tempDir.resolve("drugera_treemap.json"),
+                    Integer result = runAchillesQuery(job.getDataSource(), DRUG_ERA_SQL, tempDir.resolve("drugera_treemap.json"),
```

Comment:
if you need field of value more then one time, create new variable such as "dataSource"